apps/scorecard/.meteor/.id

```diff
@@ -0,0 +1,7 @@
+# This file contains a token that is unique to your project.
+# Check it into your repository along with the rest of this directory.
+# It can be used for purposes such as:
+#   - ensuring you don't accidentally deploy one app on top of another
+#   - providing package authors with aggregated statistics
+
+1yftguri8ke01hewq6l
```

Comment:
Does this token need to be rotated or does it not matter? It looks like this ID is shared with `upperair`.

There's reuse elsewhere between `cb-ceiling`/`surface`/etc.. so I'm inclined to think it's okay. However, the comment in the file makes it sound like they should be unique on either a per app or a per project basis.

Reply:
I wasn't aware of it until this moment, so it's probably not too important.

Reply:
Each meteor app is independent, once built, so I don't really care about this either. Pretty amazing that you saw this, though.